### DISCUSSION TITLE
Future Ready Talent - Self-Learning Courses Completion Deadline Extension!!

### DISCUSSION THREAD

**Discussion by rohithmsr** [github/india#280] [Students]
What's the deadline for the completion of self-learning courses? 220+ hours of content before Nov 25? **We have to work on our projects too and so this given time is really insufficient. It would be better if the deadline is extended.**

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
